src/plugins/lua/dmarc.lua

```diff
@@ -321,11 +321,16 @@ local function dmarc_callback(task)
         if not pct or pct == 100 or (math.random(100) <= pct) then
           task:insert_result(dmarc_symbols['quarantine'], res, lookup_domain .. ' : ' .. reason_str, dmarc_policy)
           disposition = "quarantine"
+        else
+          task:insert_result(dmarc_symbols['softfail'], res, lookup_domain .. ' : ' .. reason_str, "none", "sampled_out")
```

Comment:
disposition is set to 'none' initially here:
`local disposition = 'none'`

Should I set it to 'none' again regardless?